elementary/artifacts/runner.py

```
@@ -0,0 +1,41 @@
+
+    The internal elementary_cli project hardcodes `profile: elementary`, but
+    artifact commands need to be runnable against any working profile in the
+    user's profiles.yml (e.g. `elementary_analytics`). We inject `--profile X`
+    into every dbt invocation by wrapping `_run_command`.
+    """
+    original = runner._run_command
+
+    def patched(self, command_args, *args, **kwargs):
+        new_args = list(command_args)
+        if new_args:
+            new_args = [new_args[0], "--profile", profile] + new_args[1:]
```

Ruff (0.15.10)

[warning] Consider `[new_args[0], "--profile", profile, *new_args[1:]]` instead of concatenation

Replace with `[new_args[0], "--profile", profile, *new_args[1:]]`

(RUF005)